rhel7/kickstart/ssg-rhel7-cis-ks.cfg

```diff
@@ -59,7 +59,7 @@ network --onboot yes --device eth0 --bootproto dhcp --noipv6
 # Plaintext password is: server
 # Refer to e.g. https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw to see how to create
 # encrypted password form for different plaintext password
-rootpw --iscrypted $6$rhel6usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCtiz9c5mXbid1BzX9bmXTEZi.hCzTEXFosVBI5ng0
+rootpw --iscrypted $6$rhel7usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCtiz9c5mXbid1BzX9bmXTEZi.hCzTEXFosVBI5ng0
```

Comment:
?

Reply:
It's also in multiple other kickstarts.

Reply:
Unfortunately, yes

Reply:
this parameter seems to be the `salt` used in the password generation. I think that changing this string might break the password setting

https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw

Reply:
Ok, I've seen you removed from the pull request.

We can later change to a generic `salt`, for example:

`python -c 'import crypt; print(crypt.crypt("server", "$6$s4lt"))'`
`$6$s4lt$yASNkSe7qTgJkwSG8EohHKOKF4tyANeZhiIf02D9/U5XzUtsGMOFI7fYonLGcXN.zEOdt/rLpuLTm8Ile8.K21`